README.md

```diff
@@ -35,14 +35,14 @@ That files are unique per self-hosted (**couldn't be shared between multiple ins
 
 3. Add our Carto helm repository with the next commands:
   ```bash
-  # Add the carto-selfhosted repo.
-  helm repo add carto-selfhosted https://carto-selfhosted-charts.storage.googleapis.com
+  # Add the carto repo.
+  helm repo add carto https://helm.carto.com
 
   # Retrieve the latests version of the packages. REQUIRED before update to a new version.
   helm repo update
 
   # List the available versions of the package
-  helm search repo carto-selfhosted -l
+  helm search repo carto -l
```

Comment:
That require a change in multiple files like, for example, this file line 54.

Please look for in that repo for all `carto-selfhosted` or `carto-selfhosted/carto` to replace it in all places